cmake/CustomTargets.cmake

```diff
@@ -11,15 +11,23 @@ find_program(FORD_EXE ford
   DOC "path to the ford executable (required to generate the documentation)"
   )
 
-# copy the FORD project-file into the build directory
-configure_file(project-file.md.in project-file.md)
+# copy the FORD project-file into the build directory   UPDATE THIS!!!
+set (FORD_PROJECT_FILE "${CMAKE_BINARY_DIR}/DBCSR.md")
+configure_file(DBCSR.md "${FORD_PROJECT_FILE}")
 
 add_custom_target(doc
   COMMENT "Generating API documentation"
-  COMMAND "${FORD_EXE}" project-file.md
+  COMMAND "${FORD_EXE}" "${FORD_PROJECT_FILE}"
   VERBATIM
   )
+# EXPLAIN
+add_custom_target(symlink_doc_pages
+    #COMMAND ${CMAKE_COMMAND} -E make_directory "${CMAKE_BINARY_DIR}/docs"
+    #COMMAND ${CMAKE_COMMAND} -E create_symlink "${CMAKE_SOURCE_DIR}/docs/guide" "${CMAKE_BINARY_DIR}/docs/guide"
```

Comment:
I'd copy as long as `guide/` doesn't become too large, to keep it self-contained. Otherwise uploading to gh-pages must follow symlinks, etc.

Reply:
Sorry, I inadvertently commited these commented-out lines.
The latest version I pushed doesn't create any symlinks nor copies

Reply:
great :)